runbooks/fault-proofs.md

```diff
@@ -0,0 +1,356 @@
+# Key Handover Runbook
+
+This document describes how to generate upgrade playbooks to upgrade chains from `op-contracts/v1.3.0` (MCP) to `op-contracts/v1.X.0` (Fault Proofs).
```

Comment:
Note: determine if the target is v1.5.0 or v1.6.0. 
(I assume that's why it's given as `v1.x.0`)